<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>551</width>
           <height>80</height>
          </rect>
         </property>
         <property name="title">
          <string>System</string>
         </property>
         <widget class="QCheckBox" name="desktopNotifications">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>541</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Show Desktop Notifications</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="startOnStartup">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>541</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Start OSF Offline on Computer Startup</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>561</width>
           <height>61</height>
          </rect>
         </property>
         <property name="title">
          <string>Choose folder to Place OSF folder in </string>
         </property>
         <widget class="QPushButton" name="changeFolderButton">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>20</y>
            <width>99</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Change</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="containingFolderTextEdit">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>331</width>
            <height>31</height>
           </rect>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;/home/himanshu/somefolder/My Project&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>OSF</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>541</width>
           <height>51</height>
          </rect>
         </property>
         <property name="title">
          <string>Account</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>411</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>User name</string>
          </property>
         </widget>
         <widget class="QPushButton" name="accountLogOutButton">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>20</y>
            <width>99</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Log Out</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>541</width>
           <height>241</height>
          </rect>
         </property>
         <property name="title">
          <string>Project</string>
         </property>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>521</width>
            <height>211</height>
           </rect>
          </property>
          <property name="title">
           <string>Choose Projects to Sync With</string>
          </property>
          <widget class="QPushButton" name="changeFolderButton_2">
           <property name="geometry">
            <rect>
             <x>420</x>
             <y>100</y>
             <width>99</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
          <widget class="QTreeWidget" name="treeWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>311</width>
             <height>151</height>
            </rect>
           </property>
           <property name="toolTipDuration">
            <number>-1</number>
           </property>
           <column>
            <property name="text">
             <string>Sync</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Projects</string>
            </property>
           </column>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>115</width>
             <height>32</height>
            </rect>
           </property>
           <property name="text">
            <string>Sync All</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>20</y>
             <width>115</width>
             <height>32</height>
            </rect>
           </property>
           <property name="text">
            <string>Sync None</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>About</string>
        </attribute>
        <widget class="QTextEdit" name="textEdit_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>551</width>
           <height>251</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;This is OSF OFFLINE. Please go ahead and use it and make more software based off of it. Please and Thank You. &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;Â© Center for Open Science&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
